ui-v2/app/styles/components/tabular-collection.scss

```diff
@@ -110,6 +113,15 @@ html.template-node.template-show main table.sessions tr {
 %instances-row > * {
   width: calc(100% / 5);
 }
+%instance-paths-row > *:nth-child(1),
+%instance-paths-row > *:nth-child(5) {
+  width: calc(50% - 165px) !important;
```

Comment:
Can you explain where you're getting the numbers in this calc? Why so many !importants in this change?

Reply:
There's some detail here:

https://github.com/hashicorp/consul/blob/c2ab44412e9ecc00732cc0104854de82a0c5e77d/ui-v2/app/styles/components/tabular-collection.scss#L25-L32

In this case its 50% for each of the widest columns (the ones that might have URLs in them) columns 1 and 5. I probably just figured out visually that the port columns roughly require ~55px each, 3 port columns * 55px = 165px. I'll add some more comment here but the `55px` is probably very 'this just looked right'.

> Why so many !importants in this change?

By default tables automatically base their column widths on the number of columns, the selector to achieve this automatically is quite specific. Then, sometimes you don't want it to do it automatically and so you manually overwrite it, as is the case here. As the automatic selector is quite specific we use `!important` to make sure it gets overwritten. Also as these rules are for very specific things its highly unlikely they'll need to be overwritten again, so `!important` is a good tool for the job here.